app/wizards/placements/add_placement/steps/subject.rb

```diff
@@ -0,0 +1,24 @@
+class Placements::AddPlacement::Steps::Subject
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attribute :school
+  attribute :phase
+  attribute :subject_id
+
+  validates :school, presence: true
+  validates :phase, presence: true
+  validates :subject_id, presence: true
```

Comment:
Interestingly we don't validate that `subject_id` is valid, even though we do validate that `additional_subject_ids` are valid over in #784.

Maybe we should?

```suggestion
  validates :subject_id, presence: true,
            inclusion: { in: ->(step) { step.subjects_for_selection.ids } }
```